proposals/001-ClusterRepresentation.md

```diff
@@ -0,0 +1,305 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
+# Cluster Representation in the Model
+
+This proposal introduces the concept of _cluster_ into the model.
+
+A _downstream cluster_ is a kafka cluster that the clients connect to.   The downstream cluster can be thought of as a 'virtual' cluster or a 'facade' to
+one or more _upstream clusters_.    An _upstream cluster_ represents the physical kafka cluster.  There is a one to one correspondance between
+a _upstream cluster_ and a physical cluster in the world.
+
+There will be a mechansim to route flexibly between downstreams and upstreams.  This will allow useful topologies to built to serve different use-cases.
```

Comment:
```suggestion
There will be a mechansim to route flexibly between downstreams and upstreams.  This will allow building useful topologies to serve different use-cases.
```